### ISSUE TITLE
Time series plots - plotting "<" values or observer values with a different symbol on same plot

### ISSUE THREAD

**Issue by sbmarsh-USGS** [USGS-R/lakeReport#28]
See examples:
![image](https://cloud.githubusercontent.com/assets/10731242/16057492/ff5fefa2-323f-11e6-9a9e-1405885b04c4.png)

Green Lake observer data is different because it's taken at a depth of 0.1 instead of ours which is taken at 0.5 m below water surface.  We'll have to discuss how to differentiate the observer secchi data:
![image](https://cloud.githubusercontent.com/assets/10731242/16057539/33a4cd64-3240-11e6-98d3-5fddacaed518.png)

**Comment by lindsaycarr** [comment]
See #32 for censored values fix. Green Lake data not fixed until we figure out how to determine depth of samples (#27 is also blocked)

**Comment by sbmarsh-USGS** [comment]
Depth of sample is parameter code 00098 in QWDATA.  I don't know if that helps.

**Comment by lindsaycarr** [comment]
That helps a great deal! Thank you!

**Comment by lindsaycarr** [comment]
Is the Green Lake stuff only applicable to secchi data?

Another question: does the observer data for Green Lake have the collection code? Or can we assume any sample with a depth of 0.1 is observer? I have data where USGS_WRD also has depth values at 0.1:

![image](https://cloud.githubusercontent.com/assets/13220910/16283004/9c8938be-3890-11e6-9d9f-471c16a97298.png)

**Comment by sbmarsh-USGS** [comment]
Green Lake - yes, only secchi data.

You can assume that 0.1 depth is observer data.

**Comment by lindsaycarr** [comment]
Last question -- there are 3 Green Lake sites. Does this apply to all three or only the one in your screen shot (Green Lake at Deep Hole, 434756089020500) ??

**Comment by sbmarsh-USGS** [comment]
It applies to Green Lake Deep Hole (434756089020500) and Green Lake East End (434928088570000).

**Comment by lindsaycarr** [comment]
@sbmarsh-USGS fix for this is in - please check and close if this addresses the issue.

Lines of code that address this: https://github.com/USGS-R/lakeReport/blob/master/R/functions-qwtimeseries.R#L64-L70

```r
if(isGreenLake && isSecchi){
  usgs <- parm_data %>% filter(sample_depth != 0.1)
  observer <- parm_data %>% filter(sample_depth == 0.1) 
} else {
  usgs <- parm_data %>% filter(coll_ent_cd != "OBSERVER") 
  observer <- parm_data %>% filter(coll_ent_cd == "OBSERVER") 
}
```

**Comment by sbmarsh-USGS** [comment]
I re-ran the 2 Green Lake qwtimeseries plots after your updates and it seems to have opened a can of worms....The top two plots look good.  But for secchi...not all observer data was entered at a depth of 0.1 m, and not all USGS data even has a depth entered, so this led me to the possibility of correcting the "Collecting Agency" code to be accurate since there is currently no distinction between USGS-WRD and observer data.  Turns out, there's not an "Observer" option for that code.  If I type "OBSERVER" in that field (instead of choosing one of the designation 3-digit options, it will just pull a NULL value.  That may be our best bet - for now - in distinguishing things.  Would you be able to code it accordingly?  (See me if you have questions.)

**Comment by lindsaycarr** [comment]
@sbmarsh-USGS yes, I am able to code that. However, I am not sure we want to - that doesn't seem like the most robust solution (although I know it would be temporary). 

@ldecicco-USGS do you know if "OBSERVER" is a valid code for `coll_ent_cd`? Tried to search NWIS for valid entries for that code, but cannot seem to find any. Closest I found was `site_visit_coll_ent_cd`, but the link went nowhere http://waterdata.usgs.gov/mp/nwis/uv/help?output_formats_help

**Comment by ldecicco-USGS** [comment]
No idea. Maybe a question for the database admin...

**Comment by sbmarsh-USGS** [comment]
"OBSERVER" is not currently a valid code for "coll_ent_cd". We are requesting (to HQ) to have it created, but that won't help us in the near term.  I'm still brainstorming this one...

**Comment by sbmarsh-USGS** [comment]
All of the Green Lake observer data should now be coded "OBSERVER".
